timemachine/cpp/src/context.hpp

```diff
@@ -28,6 +28,15 @@ class Context {
 
     std::array<std::vector<double>, 3> multiple_steps_U(const int n_steps, int store_u_interval, int store_x_interval);
 
+    std::array<std::vector<double>, 2> multiple_steps_local(
+        const int local_steps,
```

Comment:
nit: `n_steps`, to align with the signature of `multiple_steps`?

Reply:
Fixed in https://github.com/proteneer/timemachine/pull/915/commits/8c4e10b69805f1531791636d369a85b2e701dbce